### DISCUSSION TITLE
希望增加 屏蔽含有“UP主的推荐”动态的功能

### DISCUSSION THREAD

**Discussion by WDonny** [the1812/Bilibili-Evolved#4702] [功能建议 / Ideas]
之前文案都是有统一的关键词还能屏蔽掉，现在每条的文案都不一样，刷着刷着蹦出条广告挺难受的。
因为这种类型的动态都含有class="bili-dyn-card-goods__mark"这个标签，感觉可以直接针对含有这个标签的动态进行屏蔽，不知道大佬百忙之中能不能新增一个这个功能。

**Comment by Geeyun-JY3** [comment]
请善用搜索，重复：#4425
有没有一种可能，这功能已经实装了，只是还在 v2.8.9 预览版：https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.8.9-preview
